akka-actor-tests/src/test/scala/akka/routing/ScatterGatherFirstCompletedSpec.scala

```diff
@@ -86,4 +89,15 @@ class ScatterGatherFirstCompletedSpec extends AkkaSpec with DefaultTimeout with
 
   }
 
+  "Scatter-gather pool" must {
+
+    "without routees should reply immediately" in {
+      val probe = TestProbe()
+      val router = system.actorOf(ScatterGatherFirstCompletedPool(nrOfInstances = 0, within = 5.seconds).props(Props.empty))
```

Comment:
Okey, sounds good. Making the 0 throw right away during construction probably does not buy much here in terms of safety for users, they'll notice anyway.